news/_posts/core-weekly/2019-06-27-coreweekly-week-25-2019.md

```diff
@@ -0,0 +1,155 @@
+* [#14221](https://github.com/PrestaShop/PrestaShop/pull/14221): Fix "14_discount/3_check_vouchers_in_FO" in full campaign. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#14282](https://github.com/PrestaShop/PrestaShop/pull/14282): Fix download in E2E tests, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+## Code changes in modules, themes & tools
+
+### Translation tools bundle
+
+
+
+### Auto upgrade
+
+* [#322](https://github.com/PrestaShop/autoupgrade/pull/322): Fix warning in 1.7 when settings file does not exist. Thank you [@jf-viguier](https://github.com/jf-viguier)
+* [#323](https://github.com/PrestaShop/autoupgrade/pull/323): Revert "Fix warning in 1.7 when settings file does not exist", by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Gamification
+
+* [#63](https://github.com/PrestaShop/gamification/pull/63): Improve UI. Thank you [@YeLnatSs](https://github.com/YeLnatSs)
+
+
+### Faceted search
+
+* [#97](https://github.com/PrestaShop/ps_facetedsearch/pull/97): Fix the display of wrong Availability count , by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#98](https://github.com/PrestaShop/ps_facetedsearch/pull/98): Bump to 3.0.5, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#99](https://github.com/PrestaShop/ps_facetedsearch/pull/99): Realease 3.0.5, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Link list
+
+* [#67](https://github.com/PrestaShop/ps_linklist/pull/67): Fix extra star in the link block form, by [@jolelievre](https://github.com/jolelievre)
+* [#68](https://github.com/PrestaShop/ps_linklist/pull/68): Merge to master for v3.0.4, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Watermark
+
+* [#15](https://github.com/PrestaShop/watermark/pull/15): [MO] Watermark - compatibility with 1.7 and ability to upload jpg, png as watermark. Thank you [@rblaurin](https://github.com/rblaurin)
+* [#16](https://github.com/PrestaShop/watermark/pull/16): Add CONTRIBUTORS.md, by [@matks](https://github.com/matks)
+* [#17](https://github.com/PrestaShop/watermark/pull/17): Version update to v2.0.0, by [@jolelievre](https://github.com/jolelievre)
+* [#18](https://github.com/PrestaShop/watermark/pull/18): Update contributors, by [@jolelievre](https://github.com/jolelievre)
+* [#19](https://github.com/PrestaShop/watermark/pull/19): Update contributors, by [@jolelievre](https://github.com/jolelievre)
+
+
+## Changes in documentation
+
+* [#279](https://github.com/PrestaShop/docs/pull/279): Reworking scale section and optimizations. Thank you [@djodjo3](https://github.com/djodjo3)
+
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @123monsite-regis, @djodjo3, @jf-viguier, @MathiasReker, @nesrineabdmouleh, @rblaurin, @sarjon, @YeLnatSs!
```

Comment:
```suggestion
Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @123monsite-regis, @djodjo3, @jf-viguier, @MathiasReker, @rblaurin!
```